core/domain/learner_progress_services.py

```diff
@@ -534,14 +744,17 @@ def remove_collection_from_incomplete_list(user_id, collection_id):
 
 
 def _remove_activity_ids_from_incomplete_list(
-        user_id, exploration_ids=None, collection_ids=None):
-    """Removes the collections and explorations from the incomplete list of the
-    user.
+        user_id, exploration_ids=None, collection_ids=None,
+        partially_learnt_topic_ids=None):
+    """Removes the collections, explorations, stories and topics
+    from the incomplete list of the learner.
```

Comment:
Docstring mentions stories, but they seem to not be included here? (Edit the docstring as discussed)

Reply:
Done.